lib/internal/util.js

```diff
@@ -271,7 +271,7 @@ function getSystemErrorName(err) {
   return entry ? entry[0] : `Unknown system error ${err}`;
 }
 
-const kCustomPromisifiedSymbol = Symbol('util.promisify.custom');
+const kCustomPromisifiedSymbol = SymbolFor('nodejs.util.promisify.custom');
```

Comment:
This should be semver-major because it impacts global scope and changes the existing symbol. Fortunately, it's not likely to break anyone, however, so I think making an exception and landing as semver-minor is fine.

Reply:
I think we’re good here, it’s reasonable to assume that the namespace of Symbols starting with `nodejs.` is owned by us.